source/query-bar.txt

```diff
@@ -146,8 +143,7 @@ maximum number of documents to return. To specify the limit, click
 Run the Query
 ~~~~~~~~~~~~~
 
-Click the :guilabel:`Apply` button to run the query and view the
-updated results.
+Click :guilabel:`Find` to run the query and view the updated results.
 
 .. note::
 
```

Comment:
You can change "click the Reset button" to just "click Reset". There is nothing else with which that could be confused.